orchagent/p4orch/tests/test_main.cpp

```diff
@@ -240,3 +241,56 @@ int main(int argc, char *argv[])
 
     return RUN_ALL_TESTS();
 }
+
+void setFlexCounterGroupParameter(const std::string &group,
+                                  const std::string &poll_interval,
+                                  const std::string &stats_mode,
+                                  const std::string &plugin_name,
+                                  const std::string &plugins,
+                                  const std::string &operation,
+                                  bool is_gearbox)
+{
+    return;
```

Comment:
why those functions are emptY ?

Reply:
The Flex counter handling logic is not invoked by p4orch. These functions are stubs for link.
We usually add mock test in mock/tests folder and add empty functions as stubs in p4orch/tests folder.